src/main/java/com/arpnetworking/tsdcore/sinks/HttpSinkActor.java

```diff
@@ -306,6 +323,31 @@ private void fireNextRequest() {
         PatternsCS.pipe(responsePromise, context().dispatcher()).to(self());
     }
 
+    private HttpResponse sendHttpRequest(
```

Comment:
Maybe add an @Override
    private HttpResponse sendHttpRequest(
            final Request request
    ) { 
      return this.sendHttpRequest(request, 0)
    }
```